cds_ils/migrator/loans/api.py

```diff
@@ -156,52 +137,51 @@ def import_loans_from_json(dump_file, raise_exceptions=False):
                 patron_pid = validate_user(record)
 
                 item = validate_item(record)
+                if not item:
+                    continue
```

Comment:
We simply skip loans without items? No issues with that?

Reply:
it prints an error in the handlers (to be cleaned for librarians)
after data is cleaned it will be raise_exception=True and it will simply fail in `validate_item`